assets/debugbar.css

```diff
@@ -157,6 +157,7 @@ a.phpdebugbar-tab.phpdebugbar-active span.phpdebugbar-badge {
   a.phpdebugbar-tab span.phpdebugbar-badge {
     display: none;
     margin-left: 5px;
+    float: right;
```

Comment:
I agree in principle, but separating these changes into two commits or PRs would be over the top. This PR is just to get the CMS working again for SS4.